packages/terralib/lua/Layer.lua

```diff
@@ -692,12 +692,15 @@ Layer_ = {
 	-- @arg data.epsg A number from the EPSG Geodetic Parameter Dataset describing a projection.
 	-- It can be used to reproject the data.
 	-- @arg data.overwrite Indicates if the exported data will be overwritten, the default is false.
+	-- @arg data.select A string table with the attribute(s) that are going to be exported. Use this only if you want to
```

Comment:
It should follow the same strategy of `CellularSpace:save`:

```
 A vector with the names of the attributes to be saved. [...] When saving a single attribute, you can use a string "attribute" instead of a table {"attribute"}.
```